main.go

```diff
@@ -36,15 +36,17 @@ var (
 		"stable": {
 			ManifestVersion: "0.1",
 			Istio: []string{
-				"https://storage.googleapis.com/knative-releases/serving/previous/v0.3.0/istio.yaml",
-				"https://storage.googleapis.com/projectriff/istio/istio-riff-knative-serving-v0-3-0-patch.yaml",
+				"https://storage.googleapis.com/knative-releases/serving/previous/v0.4.0/istio.yaml",
 			},
 			Knative: []string{
-				// NOTE: build should be in the knative-releases bucket, but is hiding in knative-nightly
-				"https://storage.googleapis.com/knative-nightly/build/previous/v0.3.0/release.yaml",
-				"https://storage.googleapis.com/knative-releases/serving/previous/v0.3.0/serving.yaml",
-				"https://storage.googleapis.com/knative-releases/eventing/previous/v0.3.0/eventing.yaml",
-				"https://storage.googleapis.com/knative-releases/eventing/previous/v0.3.0/in-memory-channel.yaml",
+				// NOTE: build release 0.4.0 is currently broken, using nightly instead
+				// https://github.com/knative/build/issues/566
+				"https://storage.googleapis.com/knative-nightly/build/latest/build.yaml",
```

Comment:
Let's go back to build@v0.2.0. There's nothing we really need in v0.4.0 other than a fix to the regression in v0.3.0 which also didn't really contain anything we urgently needed.

We can revisit with v0.4.1